modules/machineset-gcp-enabling-gpu-support.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="machineset-gcp-enabling-gpu-support_{context}"]
+= Enabling GPU support for a machine set
+
+Google Cloud Platform (GCP) Compute Engine allows users to add GPUs to VM instances. Workloads that benefit from access to GPU resources can perform better on compute machines with this feature enabled. {product-title} on GCP supports NVIDIA GPU models in the A2 and N1 machine series.
+
+.Supported GPU configurations
+|====
+|Model name |GPU type |Machine types ^[1]^
+
+|NVIDIA A100
+|`nvidia-tesla-a100`
+a|* `a2-highgpu-1g`
+* `a2-highgpu-2g`
+* `a2-highgpu-4g`
+* `a2-highgpu-8g`
+* `a2-megagpu-16g`
+
+|NVIDIA K80
+|`nvidia-tesla-k80`
+.5+a|* `n1-standard-1`
+* `n1-standard-2`
+* `n1-standard-4`
+* `n1-standard-8`
+* `n1-standard-16`
+* `n1-standard-32`
+* `n1-standard-64`
+* `n1-standard-96`
+* `n1-highmem-2`
+* `n1-highmem-4`
+* `n1-highmem-8`
+* `n1-highmem-16`
+* `n1-highmem-32`
+* `n1-highmem-64`
+* `n1-highmem-96`
+* `n1-highcpu-2`
+* `n1-highcpu-4`
+* `n1-highcpu-8`
+* `n1-highcpu-16`
+* `n1-highcpu-32`
+* `n1-highcpu-64`
+* `n1-highcpu-96`
+
+|NVIDIA P100
+|`nvidia-tesla-p100`
+
+|NVIDIA P4
+|`nvidia-tesla-p4`
+
+|NVIDIA T4
+|`nvidia-tesla-t4`
+
+|NVIDIA V100
+|`nvidia-tesla-v100`
+
+|====
+[.small]
+--
+1. For more information about machine types, including specifications, compatibility, regional availability, and limitations, see the GCP Compute Engine documentation about link:https://cloud.google.com/compute/docs/general-purpose-machines#n1_machines[N1 machine series], link:https://cloud.google.com/compute/docs/accelerator-optimized-machines#a2_vms[A2 machine series], and link:https://cloud.google.com/compute/docs/gpus/gpu-regions-zones#gpu_regions_and_zones[GPU regions and zones availability].
+--
+
+You can define which supported GPU to use for an instance by using the Machine API.
```

Comment:
Line 66, 68, and 70 in the preview just seem kind of "off" because they are by themselves. Maybe combine all three sentences into one paragraph? For example:

You can define which supported GPU to use for an instance by using the Machine API. You can configure machines in the N1 machine series to deploy with one of the supported GPU types. Machines in the A2 machine series come with associated GPUs, and cannot use guest accelerators.